contributing/contributors.md

```diff
@@ -0,0 +1,167 @@
+(contributors)=
+
+# Our contributors
+
+Curious about what our contributors have been doing? As you can see from the
+sections below (organised alphabetically), there is a huge variety of fixes,
+content updates and improvements being made across the Ubuntu Server
+documentation. The most recently submitted PRs are at the top of each table.
+
+This documentation exists because of the efforts of everyone who has taken the
+time to create, maintain and improve it. Thank you to all of you!
```

Comment:
"Thank you to all of you!" is a bit awkward phrasing.  I like awkward!
But better grammar would be either:
"Thanks go out to all of you!"
or
"Thank you all!"

However, I'd suggest even better would be to drop the expression of thanks and instead reinforce the tangible gratitude in preceding sentence, as it may read more meaningfully:

> This documentation is a testament to the valued efforts of our community members who have invested their time in its creation, maintenance, and improvement.

Reply:
That's much more elegant, I'm going to adopt that instead. Thanks @bryceharrington